README.md

```diff
@@ -26,14 +26,6 @@ containers you have running and what we can learn about them. The output from a
 Lumogon scan will be a [JSON](https://en.wikipedia.org/wiki/JSON) listing of all the
 containers found and what Lumogon could learn about them.
 
-Since JSON can be hard to read as one big unstructured message, you might want
-to use a "pretty-printer" like [jq](https://stedolan.github.io/jq/) to print the
-data in a more readable format.
-
-``` shell
-docker run --rm  -v /var/run/docker.sock:/var/run/docker.sock puppet/lumogon scan | jq
```

Comment:
I'm happy with who I am :)

![4d275c4cbade79e7831427793f9e8715_the-sexiest-am-fr-enduro-hardtail-thread-please-read-the-opening-pedantic-memes_300-274](https://cloud.githubusercontent.com/assets/83862/26250869/5dfb3260-3ca3-11e7-9260-cc79e10e718f.png)